bg_mon.c

```diff
@@ -530,9 +530,9 @@ static struct evbuffer *prepare_statistics_output(struct timeval time, system_st
 #endif
 			if (s.state == STATE_IDLEINTRANSACTION && s.idle_in_transaction_age > 0) {
 				if (s.idle_in_transaction_age < 10)
-					evbuffer_add_printf(evb, ",\"query\":\"idle in transaction %.2g\"", s.idle_in_transaction_age);
+					evbuffer_add_printf(evb, ",\"query\":%s \"idle in transaction %.2g\"", s.query, s.idle_in_transaction_age);
```

Comment:
1. `query` is already a proper JSON string (including double-quote chars in the beginning and end)
2. we need to separate the query and idle in transaction:
```json
"query":%s, "idle_in_transaction": %.2g
```